src/script/calling/CallingRepository.js

```diff
@@ -19,6 +19,8 @@
 
 import adapter from 'webrtc-adapter';
 
+import {t} from 'utils/LocalizerUtil';
```

Comment:
Please rename this to something more meaningful.

Reply:
it's quite common in translation frameworks to use one character variables.. `_` is quite popular in the Python world :)

Reply:
@lipis
True, but we are not in the Python world here. That's why we are [migrating variables to camel case](https://github.com/wireapp/wire-webapp/issues/2155) ;)

Reply:
not talking about the Python world only.. it's quite common in the general world of translation frameworks.. :) My vote is for `t` in our case :) and can be ignored in the ESLint rules

Reply:
We could use an emoji :)
What about ```💬('id')```